src/routes/NavBar.svelte

```diff
@@ -10,62 +10,125 @@
+                >
+            </li>
+            <li>
+                <a
+                    class="hidden transition-colors duration-300 ease-in-out hover:text-csi-blue lg:block"
+                    href="/events/">Events</a
+                >
+            </li>
+            <li>
+                <a
+                    class="hidden transition-colors duration-300 ease-in-out hover:text-csi-blue lg:block"
+                    href="/projects/">Projects</a
+                >
+            </li>
+        </ul>
+    </nav>
+    <div class="flex flex-row items-center gap-3 pr-1 sm:pr-4 xl:pr-32">
+        <a
+            href="mailto:lino@up-csi.org"
+            class="hidden whitespace-nowrap rounded-lg bg-csi-blue px-5 py-2 text-csi-white ring-csi-blue transition-colors duration-300 hover:bg-blue-white hover:text-csi-neutral-900 hover:shadow-inner hover:ring-2 lg:block dark:text-csi-black dark:hover:bg-black dark:hover:text-csi-white"
+        >
+            Contact Us
+        </a>
+        <button
+            class="rounded-xl border border-csi-neutral-200 p-2 transition-colors duration-200 hover:bg-csi-neutral-200 hover:shadow-inner lg:mx-2 lg:block dark:border-csi-neutral-800 dark:hover:bg-csi-neutral-800"
+            on:click={theme.toggle}
+        >
+            <Icon src={Moon} size="24px" theme="solid" class="dark:hidden " />
+            <Icon src={Sun} size="24px" theme="solid" class="hidden dark:block" />
+        </button>
+        <button
+            class="mx-2 block rounded-xl border border-csi-neutral-200 p-2 py-2 transition-colors duration-200 hover:bg-csi-neutral-200 hover:text-csi-blue lg:hidden dark:border-csi-neutral-800 dark:hover:bg-csi-neutral-800"
+            on:click={() => (mobileMenu = !mobileMenu)}
+        >
+            {#if mobileMenu}
+                <Icon src={XMark} theme="solid" size="24px" />
+            {:else}
+                <Icon src={Bars3} theme="solid" size="24px" />
+            {/if}
+        </button>
+    </div>
+</div>
 {#if mobileMenu}
-    <div class="flex h-svh flex-col bg-blue-white dark:bg-black">
-        <div class="fixed bottom-0 w-full space-y-4 pb-12">
-            <ul class="w-full font-dm text-2xl text-csi-black dark:text-csi-white">
-                <li class="m-4"><a href="/" on:click={closeMenu}>Home</a></li>
-                <li class="m-4"><a href="/about/" on:click={closeMenu}>About</a></li>
-                <li class="m-4"><a href="/people/" on:click={closeMenu}>People</a></li>
-                <li class="m-4"><a href="/events/" on:click={closeMenu}>Events</a></li>
-                <li class="m-4"><a href="/projects/" on:click={closeMenu}>Projects</a></li>
-                <li class="m-4"><button on:click={closeMenu}>Get in Touch</button></li>
-            </ul>
-            <div class="flex w-full items-center justify-center">
+    <div
+        class="fixed right-0 top-0 block h-screen w-3/4 max-w-sm bg-blue-white lg:hidden dark:bg-black"
+    >
```

Comment:
```suggestion
    <div
        class="fixed right-0 top-0 block h-screen w-full bg-blue-white sm:w-3/4 sm:max-w-sm lg:hidden dark:bg-black"
    >
```

I'm a bit unsure about the design of the mobile NavBar at the moment, At the very least, I think it should use the `w-full` for very small screens since it feels cramped otherwise.

Reply:
On this note, could we do something about the UP CSI label on the NavBar turning into two lines on smaller displays? Thanks!

Reply:
I haven't taken the very small screens into account when making this. Noted on the UP CSI logo. Would add another commit regarding this later. 

Are you fully sold on the `w-full`?  I was going for the feeling that the user is just glossing over the navigation and still connected to the page. May I suggest doing `w-full md:w-3/4` or `sm:w-3/4` whatever works best for very small screens.